native/SampleApps/RestAPIExplorer/RestAPIExplorer.xcodeproj/project.pbxproj

```diff
@@ -20,10 +34,368 @@
 		B7CEA71F20B712B1007D0564 /* RestAPI.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = B7CEA71E20B712B1007D0564 /* RestAPI.xcassets */; };
-		B7CEA73220B71C9E007D0564 /* SalesforceSwiftSDK.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7CEA73120B71C9E007D0564 /* SalesforceSwiftSDK.framework */; };
 /* End PBXBuildFile section */
 
+/* Begin PBXContainerItemProxy section */
+		B79F06D620EA930500BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06D020EA930500BC7D6F /* SalesforceSwiftSDK.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = B7DA7ABB1FCDE08E0098C1EA;
+			remoteInfo = SalesforceSwiftSDK;
+		};
+		B79F06D820EA930500BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06D020EA930500BC7D6F /* SalesforceSwiftSDK.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = B7DA7AC41FCDE08F0098C1EA;
+			remoteInfo = SalesforceSwiftSDKTests;
+		};
+		B79F06DA20EA930500BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06D020EA930500BC7D6F /* SalesforceSwiftSDK.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = B7DA7AFB1FCDE6320098C1EA;
+			remoteInfo = SalesforceSwiftSDKTestApp;
+		};
+		B79F06E420EA931200BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06DC20EA931200BC7D6F /* SmartSync.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = CE4CE2C91C0E463C009F6029;
+			remoteInfo = SmartSync;
+		};
+		B79F06E620EA931200BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06DC20EA931200BC7D6F /* SmartSync.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = 4F06AF2E1C499FD100F70798;
+			remoteInfo = SmartSyncTestApp;
+		};
+		B79F06E820EA931200BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06DC20EA931200BC7D6F /* SmartSync.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = CEAAAE55195911E600CBBFE9;
+			remoteInfo = SmartSyncTests;
+		};
+		B79F06EA20EA931200BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06DC20EA931200BC7D6F /* SmartSync.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = CEA883EE1C19160E008D871B;
+			remoteInfo = SmartSyncStatic;
+		};
+		B79F06F420EA931900BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06EC20EA931900BC7D6F /* SalesforceSDKCore.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = CE4CE26B1C0E449C009F6029;
+			remoteInfo = SalesforceSDKCore;
+		};
+		B79F06F620EA931900BC7D6F /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = B79F06EC20EA931900BC7D6F /* SalesforceSDKCore.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = 82D4642419C7C8170006BDFE;
+			remoteInfo = SalesforceSDKCoreTestApp;
```

Comment:
Why are we referencing the test targets and the static targets here? A `Swift` app can't use static libs and tests are unnecessary.

Reply:
@bhariharan Its weird the project settings doesn't show me these dependencies. They shouldn't have gotten added.